sentry-ruby/spec/support/Rakefile.rb

```diff
@@ -9,3 +9,10 @@
 task :raise_exception do
   1/0
 end
+
+task :raise_exception_without_rake_integration do
+  Sentry.init do |config|
```

Comment:
since the SDK is already initialized, this can be `Sentry.configuration.skip_rake_integration = true`